modules/apps/portal-security/portal-security-script-management-web/npmscripts.config.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```

Comment:
Hey @danvb112, could you check if is necessary change the `2000` to `2024` here?

Reply:
I was able to find more information [here](https://liferay.slack.com/archives/C1SJ64S9X/p1690365717817199?thread_ts=1689945075.217669&cid=C1SJ64S9X)